scripts/extensions/markForUser/src/common.ts

```diff
@@ -0,0 +1,9 @@
+import {ISuperdesk, IArticle} from 'superdesk-api';
+
+export function updateMarkedUser(superdesk: ISuperdesk, article: IArticle, patch: Partial<IArticle>): void {
+    superdesk.entities.article.patch(
+        article,
+        patch,
+        {patchDirectlyAndOverwriteAuthoringValues: true},
```

Comment:
when using `updateMarkedUser` for example without the item open to edit, is this option really necessary?

Reply:
In terms of code execution - no. The option will be ignored if the article is not in edit mode.

I would leave it unconditional through and assign the responsibility of figuring out when to apply it to the `patch` method.